scripts/variantstore/wdl/GvsQuickstartHailIntegration.wdl

```diff
@@ -124,17 +111,125 @@ task CreateAndTieOutVds {
+        apt-get -qq update
+        apt -qq install -y temurin-8-jdk
+
+        export PYSPARK_SUBMIT_ARGS='--driver-memory 16g --executor-memory 16g pyspark-shell'
+        pip install --upgrade pip
+        pip install hail
+
+        export TEMP_PATH=$PWD/temp
+        mkdir ${TEMP_PATH}
+
+        python3 ./vds_validation.py --vds-path ~{vds_path} --temp-path ${TEMP_PATH}
+    >>>
+    runtime {
+        # `slim` here to be able to use Java
+        docker: "gcr.io/google.com/cloudsdktool/cloud-sdk:426.0.0-slim"
+        disks: "local-disk 2000 HDD"
+        memory: "30 GiB"
+    }
+    output {
+        Boolean done = true
+    }
+}
+
 
-        python3 ./hail_join_vds_vcfs.py --vds-path ${VDS_PATH} --joined-matrix-table-path ${JOINED_MATRIX_TABLE_PATH} *.vcf.gz
+task TieOutVds {
+    input {
+        String vds_path
+        Array[File] tieout_vcfs
+        Array[File] tieout_vcf_indexes
+    }
+    parameter_meta {
+        tieout_vcfs: {
+                         localization_optional: true
+                     }
+        tieout_vcf_indexes: {
+                                localization_optional: true
+                            }
+    }
+    command <<<
+        # Prepend date, time and pwd to xtrace log entries.
+        PS4='\D{+%F %T} \w $ '
+        set -o errexit -o nounset -o pipefail -o xtrace
 
-        # Copy up the VDS
-        gcloud storage cp --recursive ${VDS_PATH} ~{vds_destination_path}
+        # Copy the Hail import and tieout scripts from Github
+        curl --silent --location --remote-name "https://raw.githubusercontent.com/broadinstitute/gatk/ah_var_store/scripts/variantstore/wdl/extract/hail_join_vds_vcfs.py"
+        curl --silent --location --remote-name "https://raw.githubusercontent.com/broadinstitute/gatk/ah_var_store/scripts/variantstore/wdl/extract/gvs_vds_tie_out.py"
+
+        # Create a manifest of VCFs and indexes to bulk download with `gcloud storage cp`.
+        touch vcf_manifest.txt
+        # This is extremely noisy and not interesting, turn off xtrace.
+        set +o xtrace
+        for file in ~{sep=' ' tieout_vcfs} ~{sep=' ' tieout_vcf_indexes}
+        do
+        echo $file >> vcf_manifest.txt
+        done
+        # xtrace back on
+        set -o xtrace
+
+        # Copy VCFs and indexes to the current directory.
+        cat vcf_manifest.txt | gcloud storage cp -I .
+
+
+        # Temurin Java 8
+        apt-get -qq install wget apt-transport-https gnupg
+        wget -O - https://packages.adoptium.net/artifactory/api/gpg/key/public | apt-key add -
+        echo "deb https://packages.adoptium.net/artifactory/deb $(awk -F= '/^VERSION_CODENAME/{print$2}' /etc/os-release) main" | tee /etc/apt/sources.list.d/adoptium.list
+        apt-get -qq update
+        apt -qq install -y temurin-8-jdk
+
+        export PYSPARK_SUBMIT_ARGS='--driver-memory 16g --executor-memory 16g pyspark-shell'
+        pip install --upgrade pip
+        pip install hail
```

Comment:
ugh all the copypasta is making me wonder if we shouldn't have a separate image for slim + Java + Hail... or undo splitting this work into separate tasks? I know I said that sounded like a good idea 😞

Reply:
hmmm, so I should just add validate vds to the task that already exists? Or are you saying forget it completely?